sklearn/linear_model/randomized_l1.py

```diff
@@ -500,10 +513,10 @@ def _make_estimator_and_params(self, X, y):
                       fit_intercept=self.fit_intercept)
         return _randomized_logistic, params
 
-    def _center_data(self, X, y, fit_intercept, normalize=False):
+    def _preprocess_data(self, X, y, fit_intercept, normalize=False):
         """Center the data in X but not in y"""
-        X, _, Xmean, _, X_std = center_data(X, y, fit_intercept,
-                                            normalize=normalize)
+        X, _, Xmean, _, X_std = _preprocess_data(X, y, fit_intercept,
```

Comment:
if you keep it Xmean -> X_mean or maybe X_offset if it can contain something else than the mean